react-ui/src/components/Header.scss

```diff
@@ -6,8 +6,8 @@
   
   .App-header-blue {
     background-color: #eae8ff;
-    height: 40px;
-    /* padding: 5px; */
+    height: 70px;
```

Comment:
I see this fix is in response to TA feedback from the demo.

Reply:
Yes it is, thank you Peter!